rero_ils/modules/patrons/api.py

```diff
@@ -230,6 +230,16 @@ def initial(self):
 
         return initial
 
+    @property
+    def formatted_name(self):
+        """Return the best possible human readable patron name."""
+        name_parts = [
+            self.get('last_name', '').strip(),
+            self.get('first_name', '').strip()
+        ]
+        name_parts = [part for part in name_parts if part]  # remove empty part
```

Comment:
Or filter

Reply:
In this case, I think list comprehension is clearer than filter (my point of view) and, if I remember correctly, list comprehension is more efficient than filter in term of memory management and speed.